### DISCUSSION TITLE
surround: mappings like vim-surround?

### DISCUSSION THREAD

**Discussion by bennypowers** [nvim-mini/mini.nvim#80] [Q&A] [answered]
:wave: I love the idea of mini (lots of small helpful opt-in modules written in lua in a single plugin), but I have years of muscle memory with `ysaw` and `rs"` etc. Can mini.surround be configured to use the same bindings as tpope/vim-surround?

**Comment by echasnovski** [comment, marked as answer]
Thanks for the feedback. That means a lot!

Sure, there is [config in the docs](https://github.com/echasnovski/mini.nvim/blob/ee7c02297cf95cba07f8bf8da3ecf99337a1c15e/doc/mini.txt#L4227) which enables behavior most similar to 'tpope/vim-surround'.
